### PR TITLE
Calculate task

### PR DESCRIPTION
**Pull request by lisie-sprawy**

### CODE REVIEW

File: module1/homework/calculate/calculate.hpp [hunk 1]
```diff
@@ -1,7 +1,21 @@
-#pragma once
 #include <string>
 
 std::string calculate(const std::string& command, int first, int second) {
     // TODO: Implement your solution here and return proper value
-    return "";
-}
+        if(command == "add"){
+        return std::to_string(first + second);
```
Comment (Kasperice) [inline module1/homework/calculate/calculate.hpp:6]: Some indentation will improve readability

File: module1/homework/calculate/calculate.hpp [hunk 2]
```diff
@@ -1,7 +1,21 @@
...
+        }
+        if(command == "multiply"){
+        return std::to_string(first * second);
+        }
+        if(command == "divide"){
+    	    if(second == 0){
+            return "Division by 0";
+            }
+        return std::to_string(first/second);
+         }
```
Comment (Kasperice) [inline module1/homework/calculate/calculate.hpp:19]: Inside that part indentation will be very important (of course it is working now), but I was wondering why `return std::to_string(first/second)` is called after `if` then I noticed it is inside.

`return std::to_string(first/second)` I saw that you are using whitespaces around operators, please add whitespaces also here

File: module1/homework/calculate/calculate.hpp (hunk 1 above)
Comment (lisie-sprawy) [reply module1/homework/calculate/calculate.hpp:6]: Yeah, thank you very much for the comments, I'll work on readability of my code!

**Comment by ziobron** [comment]
{ "requirement_ids": [1119, 1429], "points": [2, 5] }

**Pull request closed without merging**